### DISCUSSION TITLE
Getting Started

### DISCUSSION THREAD

**Discussion by wgseaton** [pfelk/pfelk#307] [Q&A] [answered]
I built Ubuntu 20.4 LTS server and ran pfelk-installer.sh and all went fine.
I web to kibana and it opens with select option.
I select elasticsearch and have data.
Using opnsense with logs pointed to server on port 5140.
There are no installed Dashboards, What do I need to do to get the pfELK dashboard?

**Comment by revere521** [comment, marked as answer]
I believe the installer script should have installed them, double check in Kibana under the menu button in the upper left, then Dashboards - otherwise there are manual install instructions here: https://github.com/pfelk/pfelk/blob/main/install/configuration.md#two-dashboards
on the config instructions page you can use to check to see if they are there and installed properly (or to install them from the repo)

**Reply by a3ilson** [reply]
The installer script should have installed them but if not you can always run the template and dashboard scripts afterwards. The only caution (issue) is if you already have logs being sent, it is advised you stop sending logs, clear your indices and then run the dashboard/template installer. Additionally, if logs were begin sent during the install, they may have created an index prior to the template/dashboard from being installed which will yield the same behavior.

Scripts are located at the following: 
https://github.com/pfelk/pfelk/tree/main/etc/pfelk/scripts
